shopinvader_cart_expiry/views/shopinvader_backend.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="shopinvader_backend_form_view">
+        <field name="name">shopinvader.backend.form (in shopinvader_cart_expiry)</field>
+        <field name="model">shopinvader.backend</field>
+        <field name="inherit_id" ref="shopinvader.shopinvader_backend_view_form"/>
+        <field name="priority" eval="90"/>
+        <field name="arch" type="xml">
+            <page name="sale" position="inside">
+                <group>
+                    <group string="Cart Expiry Management">
```

Comment:
better to add a name to the group to make it compat w/ v12. Not a blocker

Reply:
@simahawk  You are right. Just did it. Thanks